package.json

```diff
@@ -36,6 +36,7 @@
     "react-dom": "^18",
     "react-hook-form": "^7.53.1",
     "sass": "^1.80.4",
+    "swiper": "^11.1.15",
```

Comment:
we can take swiper but splidejs is better, various animations and splidejs is not Russian like swiper

Reply:
I've changed it, but I still need to customise the arrows and pagination

Reply:
Done